c_emulator/riscv_sim.c

```diff
@@ -416,22 +431,28 @@ void init_spike(const char *f, uint64_t entry, uint64_t ram_size)
   s = tv_init(isa, ram_size, 1);
   if (tv_is_dirty_enabled(s) != rv_enable_dirty_update) {
     mismatch = true;
-    fprintf(stderr, "inconsistent enable-dirty-update setting: spike %s, sail %s\n",
+    fprintf(stderr,
+            "inconsistent enable-dirty-update setting: spike %s, sail %s\n",
             tv_is_dirty_enabled(s) ? "on" : "off",
             rv_enable_dirty_update ? "on" : "off");
   }
   if (tv_is_misaligned_enabled(s) != rv_enable_misaligned) {
     mismatch = true;
-    fprintf(stderr, "inconsistent enable-misaligned-access setting: spike %s, sail %s\n",
-            tv_is_misaligned_enabled(s) ? "on" : "off",
-            rv_enable_misaligned        ? "on" : "off");
+    fprintf(
+        stderr,
```

Comment:
Ditto

Reply:
I can't figure out a workaround for these. It looks like aligning function arguments to the open paren forces the one-per-line formatting if it encounters a long string literal. And manually line-breaking it doesn't seem to help.